creusot/tests/should_fail/final_borrows.rs

```diff
@@ -11,18 +11,20 @@ pub fn not_final_borrow<T: Copy>(bor: &mut T) {
 
 pub fn store_changes_prophecy<T: Copy>(bor: &mut T) {
     let b1 = &mut *bor;
+    let b1_snap = snapshot!(b1);
     let val = *b1;
```

Comment:
Wow, so this use of `b1` *uninitializes* it?

Reply:
Ok, I was _really_ not thinking... none of those are actually uninitialized, except for `bor` in `move_place_without_deref` (which makes sense). I'm fixing this now.